byteparse/shared/src/test/scala/byteparse/classparse/ClassTests.scala

```diff
@@ -0,0 +1,1180 @@
+          "IAAhkWtgAEsgACGRe2AAQQCrwKOhgEvQAhOhkQZLwEOhoGvApZAxEjKE9ZBBED6E9ZBREFOU86G" +
+          "we9ACFZAxIiU1kEEiNTWQUSJFNZBhIlUzocBrwEWQMEVFkEA1RZBQNUOh2yAAK7ACZZtwAnEii2" +
+          "ACkZGAMutgAqtgArtgAEGRgEBE+yAAK7ACZZtwAnEiy2ACkZGAQutgAqtgArtgAEsgACEi22AAQ" +
+          "ENh4FNh+yAAK7ACZZtwAnEi62ACkVHhUfYLYAKrYAK7YABLIAArsAJlm3ACcSL7YAKRUfFR5ktg" +
+          "AqtgArtgAEsgACuwAmWbcAJxIwtgApFR8VHmi2ACq2ACu2AASyAAK7ACZZtwAnEjG2ACkVHhUfb" +
+          "LYAKrYAK7YABLIAArsAJlm3ACcSMbYAKRUehxUfh2+2ADK2ACu2AASyAAISM7YABLIAAhI0tgAE" +
+          "sgACEjW2AASyAAISNrYABLIAAhI3tgAEsgACEji2AASyAAISObYABLIAAhI6tgAEsgACEju2AAS" +
+          "yAAISPLYABAM2ILIAAhI9tgAEsgACFSCEIAG2AD6yAAKEIAEVILYAPrIAAhUghCD/tgA+sgAChC" +
+          "D/FSC2AD6yAAISP7YABBAKNiEVIRAKoAAOsgACEkC2AASnAB0VIRAKpAAOsgACEkG2AASnAAuyA" +
+          "AISQrYABAM2IhUiEGSiABGyAAIVIrYAPoQiAaf/7rIAArsAJlm3ACcSQ7YAKRUitgAqtgArtgAE" +
+          "AzYjsgACFSO2AD6EIwEVIxBkof/xsgACuwAmWbcAJxJEtgApFSO2ACq2ACu2AAQDNiQVJBAKogA" +
+          "RsgACFSS2AD6EJAGn/+4DNiQVJBAKogAoAzYlFSUQCqIAGBUkCKAADBUlCKAABqcAD4QlAaf/54" +
+          "QkAaf/1xAJvApZAwRPWQQFT1kFBk9ZBgdPWQcIT1kIEAZPWRAGEAdPWRAHEAhPWRAIEAlPOiQZJ" +
+          "DolGSW+NiYDNicVJxUmogAYGSUVJy42KLIAAhUotgA+hCcBp//nBjYlFSWqAAAALgAAAAEAAAAD" +
+          "AAAAGQAAACAAAAAnEkU6JqcAFRJGOianAA4SRzompwAHEkg6JrIAArsAJlm3ACcSSbYAKRkmtgA" +
+          "ptgArtgAEEko6JxknOigCNikZKLYAS6sAAABOAAAAAwAADcEAAAAxAAHS5wAAACEGLexoAAAAQR" +
+          "koEky2AE2ZACYDNimnACAZKBJOtgBNmQAWBDYppwAQGSgSSrYATZkABgU2KRUpqgAAADoAAAAAA" +
+          "AAAAgAAABkAAAAkAAAAL7IAAhJPtgAEpwAzsgACElC2AASnACiyAAISUbYABKcAHbIAArsAJlm3" +
+          "ACcSUrYAKRkntgAptgArtgAECDYoFSgQCqIACBJTpwAFElQ6KbIAAhkptgAEElW4AFZXEHu4AFd" +
+          "XsQAAAAIAXAAAAb4AbwAAAAgACAAJABAACgAYAAsAIAAMADYADwA5ABEAPwATAEMAFABIABUATQ" +
+          "AWAFEAFwBWABgAWQAZAFwAGgBgAB0AZQAeAHAAHwB9ACAAiAAhAIwAIgCQACMAlAAkAJwAJQCkA" +
+          "CYArAAnALIAKAC4ACkAvgAqANUAKwDvACwBAAAtARwALgEhAC8BPQAwAUsAMQFoADIBhQAzAaIA" +
+          "NAG/ADUB3gA2AeYANwHuADgB9gA5Af4AOgIGADsCDgA8AhYAPQIeAD4CJgA/Ai4AQAIxAEECOQB" +
+          "CAkQAQwJPAEQCWgBFAmUARgJtAEcCcQBIAngASQKDAEoCigBLApUATQKdAE8CoABQAqcAUQKvAF" +
+          "ICtQBUAs8AVQLSAFcC2gBYAt0AWQLkAFoC/gBbAwgAXAMQAFsDFgBfAyAAYAMqAGEDNgBiAzkAY" +
+          "AM/AF8DRQBmA3YAZwOQAGgDmABnA54AagOhAGwDvABtA8AAbgPDAG8DxwBwA8oAcQPOAHID0QBz" +
+          "A9UAdgPvAHcD8wB4BGgAegRwAHsEcwB9BHsAfgR+AIAEhgCBBIkAgwSjAIYEpgCHBLYAiAS+AIk" +
+          "ExACKBMoAiwBeAAAA2QAf/wKDAB8HAF8BAQEBAQEEAgMBAQEAAwcAYAcAYQcAYQcAYgcAYgcAYg" +
+          "cAYwcAZAcAZQcAYwcAZAcAZQEBAQEAABEH/AACART8ABwB/AAuAfoAFPwAAgH8AAkBFfoABfoAB" +
+          "f8APAAlBwBfAQEBAQEBBAIDAQEBAAMHAGAHAGEHAGEHAGIHAGIHAGIHAGMHAGQHAGUHAGMHAGQH" +
+          "AGUBAQEBAQEHAGMHAGMBAQAA+AAb/AAdAQYGBvwAAwcAYv4ASgcAYgcAYgEPDwwaCgoK+QAZ/AA" +
+          "OAUEHAGIAAQBmAAAAAgBn").toByteArray
+        /* CodeTest.class from compiled class CodeTest.java */
```

Comment:
Could we assert things about the constant pool of this class? We don't need to check every since entry, but we should at least spot-check some of the values to make sure some of them e.g. `100000L 2.71828 "Do you want to add a tab?\tNo Problem!" "\n->Control Structures" 1337` which we can see in the source code are present in the constant pool.